src/demo-app/datepicker/datepicker-demo.ts

```diff
@@ -97,3 +98,18 @@ export class CustomHeader<D> implements OnDestroy {
         this._dateAdapter.addCalendarYears(this._calendar.activeDate, 1);
   }
 }
+
+// Custom header component for datepicker using standard header
+@Component({
+  moduleId: module.id,
+  selector: 'custom-header2',
+  template: `
+  <div>Custom element</div>
```

Comment:
@mmalerba Prepending elements may break the design (missing space at the bottom of the calendar body). Can this be solved with CSS?

Reply:
Yeah that's a good point. I think for now if people are overriding the header they can just add CSS to make the popup taller as well. It would be good to see if we can change the Material CSS to make it work automatically though

Reply:
Ok!